src/core/include/databases.h

```diff
@@ -17,16 +18,18 @@ class GeomStorage {
 	void Clear() { shapes_.clear(); }
 
  private:
+	// we assume that shape id equual to plase of shape in the array.
```

Comment:
Bad assuming. Need to be fixed in the future. Possible and obvious solution is to make getters in such classes (`GetSmth`, to determinate complexity of getters), where it finds nedded object in corresponding container by stored id